### DISCUSSION TITLE
PDF file format in Wiki.

### DISCUSSION THREAD

**Discussion by MaksGill** [outline/outline#4684] [General]
Guys, can someone tell you if you managed to upload a PDF file to Outline? If so, how did you do it?

**Comment by ghost** [comment]
It works using the '/file attachment' command or by dragging and dropping the file to where the cursor in blinking.

**Reply by MaksGill** [reply]
When dragging a pdf file manually or through a tab with "+" functions. The message "Upload failed" comes out like this. What's going wrong?

**Reply by tommoor** [reply]
You can add a PDF as an attachment to a document through the above method. There is no way to import a PDF and convert it to a document though. Your best bet is to copy/paste the content in that case
